java/src/jmri/jmrit/signalling/AddEntryExitPairPanel.java

```diff
@@ -135,10 +135,14 @@ private void addButton() {
 
     private void autoDiscovery() {
         if (!InstanceManager.getDefault(jmri.jmrit.display.layoutEditor.LayoutBlockManager.class).isAdvancedRoutingEnabled()) {
-            int response = JOptionPane.showConfirmDialog(null, Bundle.getMessage("EnableLayoutBlockRouting"));
+            int response = JOptionPane.showConfirmDialog(null,
```

Comment:
It would be preferable to merge the multiple resource bundles in this package into a single resource bundle to replacing Bundle.getMessage() with the direct loading of the ResourceBundle. Since the key "EnableLayoutBlockRouting" is defined identically in multiple packages, using Bundle.getMessage() will allow the eventual replacement of multiple strings with the same meaning with a single translation.